app/models/digital_object/asset.rb

```diff
@@ -29,6 +29,15 @@ class DigitalObject::Asset < DigitalObject
 
   attr_accessor :skip_resource_request_callbacks
 
+  def generate_label
```

Comment:
```suggestion
  def generate_display_label
```
How would you feel about changing all occurrences of generate_label to generate_display_label?

Reply:
Done!